src/main/groovy/life/qbic/utils/DataSetValidationException.groovy

```diff
@@ -0,0 +1,41 @@
+package life.qbic.utils
+
+import org.everit.json.schema.ValidationException
+
+/**
+ * Exception that shall be thrown when a dataset validation failed.
+ *
+ * This exception class wraps the underlying <code>org.everit.json.schema
+ * .ValidationException</code> class, to encapsulate this third party dependency.
```

Comment:
```suggestion
 * <p>This exception class wraps the underlying <code>org.everit.json.schema
 * .ValidationException</code> class, to encapsulate this third party dependency.</p>
```
How do we use this class in case we decide to not use the ValidationException? I am not sure that I understand why we have this class.

Reply:
This class is an adapter class. I don't want to propagate everit dependencies in our code base, thats why I wrapped it.

Reply:
I see that point as well. The question is how effective the wrapping is in this case. What would happen if we switch out the dependency? Is our wrapper designed in a way that it could use other libraries?